Monika-After-Story/MonikaModDev
# Sakuya outfit

# Key Changes

**new spritepack** giftname is `sakuyaizayoicosplay`

* sakuya izayoi clothes
* bunbraid hair style (mix of bun and ponytail braid)
* green bow strand (this one is not given a generic name since we're not looking to add acs dependent on braided-strand hairstyles)
* maid headband (only allowed on the bunbraid hair style)
* `braidstrand` exprop added to `ponytail-braid`

### Addititional sprite progpoint functions:
* `_acs_wear_if_found` - wears an ACS given its name if the ACS exists
* `_acs_remove_if_found` - removes an ACS given its name if the ACS exists

TODO:
- [x] thumbnail
- [x] selector dlg for clothes
- [x] gift reaction dialogue

# Testing
* verify sakuya outfit looks ok
* verify that the headband only can be worn with the bunbraid hair style (everything else will take off the headband)
* verify that the bow strand only can be worn with braided strand hairstyles (bunbraid and ponytail-braid, for the latter you will need the updated json from here)

## Other testing notes:
* the headband is mutually exclusive with the miku headset. It is **not** exclusive with ribbons as the headband's position is clearly in front of a ribbon's location.
* in the clothes selector, both the headband and the bow strand will be taken off only if outfit mode is enabled during the removal (or if the desired clothes/hair requires the acs to be removed).  Miku has also been changed to reflect this behavior.
